Userland/Libraries/LibGfx/Color.cpp

```diff
@@ -310,6 +310,35 @@ Optional<Color> Color::from_string(StringView string)
     return Color(r.value(), g.value(), b.value(), a.value());
 }
 
+static float mix(float x, float y, float a)
+{
+    return x * (1 - a) + y * a;
+}
```

Comment:
We already have `mix` in `AK/StdLibExtras.h`

Reply:
Ah thanks, missed that :)